wcfsetup/install/files/ts/WoltLabSuite/Core/Form/Builder/Manager.ts

```diff
@@ -0,0 +1,175 @@
+/**
+ * Manager for registered Ajax forms and its fields that can be used to retrieve the current data
+ * of the registered forms.
+ *
+ * @author  Matthias Schmidt
+ * @copyright 2001-2020 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Form/Builder/Manager
+ * @since 5.2
+ */
+
+import * as Core from "../../Core";
+import * as EventHandler from "../../Event/Handler";
+import Field from "./Field/Field";
+import DependencyManager from "./Field/Dependency/Manager";
+import { FormBuilderData } from "./Data";
+
+const _fields = new Map<string, Map<string, Field>>();
+const _forms = new Map<string, HTMLElement>();
+
+const FormBuilderManager = {
+  /**
+   * Returns a promise returning the data of the form with the given id.
+   */
+  getData(formId: string): Promise<FormBuilderData> {
```

Comment:
Export these as bare functions and not as part of a bundled object with a legacy `export =`.